www/runtest.php

```diff
@@ -3283,27 +3284,29 @@ function ReportAnalytics(&$test, $testId)
 }
 
 function loggedOutLoginForm(){
-  $ret = '<ul class="testerror_login"><li><a href="/saml/login.php">Login</a></li>';
-  $reg = GetSetting('saml_register');
-  if ($reg) {
-    $ret .= "<li><a class='pill' href='$reg' onclick=\"try{if(_gaq!=undefined){_gaq.push(['_trackEvent','Outbound','Click','Signup']);}}catch(err){}\">Sign-up</a></li>";
-  }
-  $ret .= "</ul>";
+    $ret = <<<HTML
+<ul class="testerror_login">
+    <li><a href="/login">Login</a></li>
+    <li><a class='pill' href='/signup'>Sign-up</a></li>
+</ul>
+HTML;
+
   return $ret;
 }
 
 function loggedInPerks(){
-  $msg = '<ul class="testerror_loginperks">';
-  $msg .= '<li>Access to 13 months of saved tests, making it easier to compare tests and analyze trends.</li>';
-  $msg .= '<li>Ability to contribute to the <a href="https://forums.webpagetest.org/">WebPageTest Forum</a>.</li>';
-  $msg .= '<li>Access to upcoming betas and new features that will enhance your WebPageTest experience.</li>';
-  $msg .= '</ul>';
+    $msg = <<<HTML
+<ul class="testerror_loginperks">
```

Comment:
so my linter is gonna be mad about this but it'll get fixed next time I touch this